.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9]
```

Comment:
I also updated this as GitHub Actions no longer supports Python 3.7 on ubuntu-latest